blogs/2016/03/11/ExtensionsRoundup.md

```diff
@@ -0,0 +1,80 @@
+---
+Order: 4
+TOCTitle: Extensions Roundup
+PageTitle: Visual Studio Code Extensions
+MetaDescription: New, useful, and interesting extensions in Visual Studio Code. 
+Date: 2016-03-11
+ShortDescription: New, useful, and interesting extensions in Visual Studio Code.
+Author: Wade Anderson
+---
+
+# VS Code Extensions
+
+Three months have passed since we open sourced VS Code and introduced the [extensions API](http://code.visualstudio.com/docs/extensionAPI/vscode-api). 
+The [Extensions Marketplace](https://marketplace.visualstudio.com/VSCode) has over 850 extensions. Many new languages and frameworks are now supported.
```

Comment:
Do you want to call out some examples:
Something like "Many new languages (Go, PowerShell, Python) and frameworks (Apache Cordova, Chrome browser) are not supported."
Might as well call out the popular extensions.